Moya-Argo.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "Moya-Argo"
-  s.version          = "2.0.0"
+  s.version          = "3.0.0"
```

Comment:
should this be a major version bump if both of the dependencies are patch releases? (at least as far as version numbers go)